<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>个人工作与任务管理系统</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QWidget" name="leftWidget" native="true">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>任务分类</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="categoryList">
         <property name="minimumSize">
          <size>
           <width>180</width>
           <height>0</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QPushButton" name="addCategoryBtn">
           <property name="text">
            <string>添加分类</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="deleteCategoryBtn">
           <property name="text">
            <string>删除分类</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="centerWidget" native="true">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>查询：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="searchEdit"/>
         </item>
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>优先级：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="priorityCombo"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTableView" name="taskTable"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QPushButton" name="addTaskBtn">
           <property name="text">
            <string>添加任务</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="editTaskBtn">
           <property name="text">
            <string>编辑任务</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="deleteTaskBtn">
           <property name="text">
            <string>删除任务</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="markCompletedBtn">
           <property name="text">
            <string>标记完成/未完成</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="exportExcelBtn">
           <property name="text">
            <string>导出Excel</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="exportPdfBtn">
           <property name="text">
            <string>导出PDF</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="showGanttBtn">
           <property name="text">
            <string>查看甘特图</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
